<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>20</y>
      <width>741</width>
      <height>421</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tabHellow">
     <attribute name="title">
      <string>Tab Hellow</string>
     </attribute>
     <widget class="QPushButton" name="pushButtonHellow">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>HelloText</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelHellow">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>26</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabelForHellow</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabCalculator">
     <attribute name="title">
      <string>Tab calculator</string>
     </attribute>
     <widget class="QLineEdit" name="lineEditNum1">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>60</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditNum2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>100</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelNum1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>First number:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelNum2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Second number:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSum">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>180</y>
        <width>120</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>sum</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSubtraction">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>230</y>
        <width>120</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>subtraction</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonMultiplication">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>180</y>
        <width>120</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>multiplication</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonDivision">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>230</y>
        <width>120</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>division</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>300</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>The result is:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditResults">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>300</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
